### ISSUE TITLE
Make "Extra Attributes" writable via the API

### ISSUE THREAD

**Issue by ChrisGrundemann** [nautobot/nautobot-plugin-bgp-models#108]
### Environment
* Nautobot version:  v1.5.9
* nautobot-bgp-models version:  0.7.0b1

### Proposed Functionality
The Extra Attributes field is writable in the GUI, and I want it to be writable in the API as well. This should apply to all objects with the Extra Attributes field.

Today I am unable to write to the Extra Attributes field in the BGP Models plugin, e.g:
```
    nb = pynautobot.api(url=nauto_url, token=nauto_token)
    peergroups = nb.plugins.bgp.peer_groups

    PeerGroup_config = {
        "name": "TEST PeerGroup",
        "description": "this is a test - will delete later",
        "enabled": True,
        "routing_instance": "91de2bdd-64dc-4359-8197-426e0577ace7",
        "extra_attributes": {
            "test_attribute_01": "test",
            "test_attribute_02": 666
        },
    }

    new_PG = peergroups.create(**PeerGroup_config)
```
Creates the record, but with a blank (null) Extra Attributes field.

Also, when I try to update this way:
```
    nb = pynautobot.api(url=nauto_url, token=nauto_token)
    peergroup = nb.plugins.bgp.peer_groups.get(name="TEST PeerGroup")

    peergroup.extra_attributes["test_attribute_01"] = "Not a test"
    peergroup.save()
    print(peergroup.extra_attributes["test_attribute_01"])
```
I get an error: `TypeError: 'NoneType' object does not support item assignment`

### Use Case
In order to use the BGP Models plugin in anything other than a greenfield deployment, a user must import the existing network configurations into Nautobot. Doing this manually for hundreds or thousands of BGP peers (true even in a small ISP network) is hugely burdensome - for example, even if it only takes 2 minutes per record (conservative) it would take almost 7 hours to input just 200 records.

In order to quickly sync Nautobot BGP Models data with the network, these fields need to be writable not just in the GUI but also in the API.

**Comment by mzbroch** [comment]
This issue has already been addressed and is fixed in 0.7.1 release. Please re-test.

**Comment by ChrisGrundemann** [comment]
Looks like this may have been fixed in pull #99 - will test this week to be sure

**Comment by ChrisGrundemann** [comment]
FYI - Upgrade worked - this is indeed fixed in 0.7.1 -- thanks all!
